test/Microsoft.AspNetCore.Sockets.Tests/WebSocketsTests.cs

```diff
@@ -29,7 +31,8 @@ public class WebSocketsTests
             using (var applicationSide = ChannelConnection.Create<byte[]>(transportToApplication, applicationToTransport))
             using (var feature = new TestWebSocketConnectionFeature())
             {
-                var ws = new WebSocketsTransport(new WebSocketOptions(), transportSide, connectionId: string.Empty, loggerFactory: new LoggerFactory());
+                var connectionContext = new DefaultConnectionContext(string.Empty, null, null);
```

Comment:
This isn't required, we should just pass the connection id and the enum directly.